app/controllers/herbarium_records_controller.rb

```diff
@@ -256,7 +283,7 @@ def normalize_parameters
     @herbarium_record.notes = @herbarium_record.notes.to_s.strip
   end
 
-  def validate_herbarium_name!
+  def validate_herbarium_name! # rubocop:disable Metrics/AbcSize
```

Comment:
Good on you!  Fight the Man! :)

Reply:
I've got those in a salt shaker right here on my desk. 
But i am breaking down a lot of methods, too.